tools/rush-update/src/main.ts

```diff
@@ -1,11 +1,12 @@
 import { updatePackageFiles } from './update-npm';
-import { getProjectFolders, updateRushShrinkwrapFile } from './rush';
-import { commitCurrentChangesToBranch, pushToRemoteBranch } from './git';
+import { getProjectFolders, updateRushShrinkwrapFile, generateRushChangeFiles } from './rush';
+import { switchToNewBranch, commitChanges, pushToRemoteBranch } from './git';
 import { createPullRequest } from './github';
 import { NCUParams } from 'npm-check-updates';
 
 export const DEFAULT_BRANCH = 'update-npm-dependencies';
```

Comment:
Just noticed that the branch name is constant, do you think it would work better with a dynamic branch like `update-npm-dependencies-2019-07-08`, otherwise every PR needs to be reviewed or the following ones will fail, even during a weekend.

Reply:
maybe we should fine tune it a bit (to run only once on the weekend)
but anyway, I'm for it failing at some point if a PR/branch already exists